tests/test-sources/plugins/by-name/conform-nvim/default.nix

```diff
@@ -4,6 +4,18 @@
     plugins.conform-nvim.enable = true;
   };
 
+  all-formatters =
+    let
+      allFormatters = import ../../../../../generated/conform-formatters.nix;
```

Comment:
Does the test enforce that all formatters listed here have an attribute in `plugins/by-name/conform-nvim/formatter-packages.nix`? I can't see any assertions for missing attributes but I'm probably mis-understanding how the test works.